### ISSUE TITLE
Fix broken Mochawesome videos

### ISSUE THREAD

**Issue by CBonade** [department-of-veterans-affairs/va.gov-team#34522]
During our refactoring of the Mochawesome ingestion mechanisms, we unintentionally broke videos loading properly in the Mochawesome reports. These videos are invaluably helpful for anyone that wants to troubleshoot their failing tests and should be fixed as soon as we can.

**Comment by CBonade** [comment]
This is now merged and fixed. The culprit was both the video artifact download, and subsequent upload of videos to S3 not jiving with our directory restructure we implemented in our refactoring of the ingestion mechanism. Additionally, some work was needed to be done to alter how we were rewriting the video URLs to be found appropriately at the S3 level. All set and confirmed with a deliberately failed test that the video displayed properly.
